GOVERNANCE.md, CONTRIBUTING.md: add inclusive language blurb (#4516)
tldr-pages/tldr: GOVERNANCE.md

   Remember, it's very hard to convey meaning on a purely written medium,
   especially between people from different cultures, technical backgrounds,
   English proficiency levels, etc.

>    Where possible, we use inclusive language, but sometimes we have kept the
>    original to preserve the original meaning of the text. We are merely
>    *documenting* here, not taking a political stance. If a tool attributes
>    any particular meaning to a particular term (i.e. master/slave) we
>    document it as is.

Reviewer: I think this can be shortened, we are talking about *governance* here, no need to go into the details. That kind of longer explanation is better suited for `CONTRIBUTING.md`.

```suggestion
   Where possible, we use inclusive language, but sometimes we have kept the
   original terminology to preserve the meaning of the text. We are merely
   *documenting* here, not taking any political stance.
```

   Remember, it's very hard to convey meaning on a purely written medium,
   especially between people from different cultures, technical backgrounds,
   English proficiency levels, etc.
   Where possible, we use inclusive language, but sometimes we have kept the
   original terminology to preserve the meaning of the text. We are merely

>    *documenting* here, not taking any political stance.

Reviewer 2: I agree with adding a note about this here, but the current phrasing is unnecessarily defensive. Also, these rules are about communicating with one another, as members of our community, not about the content of the pages. IMO we can edit this to something like the following:

```
Use inclusive language whenever possible and reasonable — for example,
[avoid assuming gender](https://en.wikipedia.org/wiki/Singular_they),
and refrain from racially or ethnically charged terms.
```
